velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -24,10 +24,13 @@ namespace facebook::velox::exec::test {
 /// Query runner that uses reference database, i.e. DuckDB, Presto, Spark.
 class ReferenceQueryRunner {
  public:
-  enum class RunnerType { kPrestoQueryRunner, kDuckQueryRunner };
+  explicit ReferenceQueryRunner(memory::MemoryPool* rootPool)
```

Comment:
I assume this doesn't have to be the root pool. Any aggregate pool would suffice. If so, consider renaming to aggregatePool.

It would be nice to clarify that this pool is expected to be used to allocate memory needed for vectors produced by 'execute' methods.

Reply:
Thanks for the suggestion. Renamed the variable aggregatePool and added documentation.